### PR TITLE
Update peers.txt added Everstake node peer

### PR DESCRIPTION
**Pull request by bo-everstake**
## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: celestia/peers.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+87af580078c80c630625db2360b3dc19483d29cd@185.182.194.163:26656
```
Comment (evan-forbes) [inline celestia/peers.txt:1]: please fix conflict by submitting the peer info in a separate file in this directory named as `your-validator.txt`

**Review by evan-forbes — CHANGES_REQUESTED** [review]

File: celestia/peers.txt [hunk 2]
```diff
@@ -1,17 +0,0 @@
...
-1b92a439555a19ee6edea85b184abb6873757dae@51.89.173.96:26656
-ea9d858371ba86ae8f9c5b292c105329029c9efd@5.199.174.19:26656
-5bc86308e2b3730ce611abf622c8be752780f934@5.199.172.53:26656
-acca7837e4eb5f9dc7f5a94ed1d82edda6931ff8@rpc-celestia.pops.one:26656
-f5bce79feec9068154e7f50fe8db7a8191a58663@82.220.38.204:10056
-629c36a551ee1a3c8bb3ecee6e00a9977a60e53e@65.108.46.248:56656
-d0c530290267e1c539f27061c9446d5042eaf1ca@161.97.180.20:26656
-a41b4bc451b1b71d537aa1018226e08d7fa7e44e@5.255.77.44:26656
-2f05b07cde80c5b08ee0159658f7977c7765f966@148.113.17.55:26756
-5001de72be39622c9dc34f2117eccc3f3fca8a7a@celestia-mainnet-01.twinstake.io:26756
```
Comment (evan-forbes) [inline celestia/peers.txt:17]: to clarify slightly, we just should not modify this file at all. thanks

Comment (bo-everstake) [reply celestia/peers.txt:17]: returned back peers.txt

File: celestia/peers.txt [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
+-1b92a439555a19ee6edea85b184abb6873757dae@51.89.173.96:26656 
+-ea9d858371ba86ae8f9c5b292c105329029c9efd@5.199.174.19:26656 
+-5bc86308e2b3730ce611abf622c8be752780f934@5.199.172.53:26656 
+-acca7837e4eb5f9dc7f5a94ed1d82edda6931ff8@rpc-celestia.pops.one:26656 
+-f5bce79feec9068154e7f50fe8db7a8191a58663@82.220.38.204:10056 
+-629c36a551ee1a3c8bb3ecee6e00a9977a60e53e@65.108.46.248:56656 
+-d0c530290267e1c539f27061c9446d5042eaf1ca@161.97.180.20:26656 
+-a41b4bc451b1b71d537aa1018226e08d7fa7e44e@5.255.77.44:26656 
+-2f05b07cde80c5b08ee0159658f7977c7765f966@148.113.17.55:26756 
+-5001de72be39622c9dc34f2117eccc3f3fca8a7a@celestia-mainnet-01.twinstake.io:26756
```
Comment (evan-forbes) [inline celestia/peers.txt:17]: @bo-everstake to clarify we need to make sure that we're not modifying this file. thanks!

**Pull request closed without merging**